src/types.ts

```diff
@@ -699,6 +701,23 @@ export interface KaboomCtx<
      * @group Components
      */
     shader(id: string, uniform?: Uniform | (() => Uniform)): ShaderComp;
+    /**
+     * Enable timer related functions like wait(), loop(), tween() on the game object.
+     *
+     * @example
+     * ```js
+     * const obj = add([
+     *     text(""),
+     *     textInput(),
+     * ])
+     *
+     * obj.hasFocus = false
+     * debug.log(obj.text) // oh no i cant see my new text since it was disabled
```

Comment:
You just can't input text right? You can still see it.

Reply:
yes, when hasFocus is false then you cant put new text, you can still see all the old text